### ISSUE TITLE
dbt version 0.16 issue

### ISSUE THREAD

**Issue by dubravcik** [mikaelene/dbt-sqlserver#16]
Hi, I upgraded dbt from 0.15.2 to 0.16.1 and dbt-sqlserver stopped working and I am confused by the error. I haven't found any duplicates in schema_test.sql

```
Running with dbt=0.16.1
Encountered an error:
Compilation Error
  dbt found two macros with the name "test_unique" in the namespace "dbt". Since these macros have the same name and exist in the same namespace, dbt will be unable to decide which to call. To fix this, change the name of one of these macros:
  - macro.dbt_sqlserver.test_unique (macros\schema_tests.sql)
  - macro.dbt_sqlserver.test_unique (macros\schema_tests.sql)
```

thanks

**Comment by mikaelene** [comment]
Hi! This adapter doesn’t support Dbt 0.16.x. Install dbt 0.15.x to solve this. I have tried to upgrade but changes in dbt internal makes it impossible to make tests work. One way is to release an adapter without support for tests. This will be fixed in 0.17 of dbt so I think I will pass on 0.16 for the adapter.

**Comment by mikaelene** [comment]
I have submitted a fix for this in dbt core but I think it will land in 0.17.   https://github.com/fishtown-analytics/dbt/pull/2414 . Closing this. Hopefully all will be good in 0.17
